### ISSUE TITLE
Initialization totally fails for versions 1.1.0 and 1.2.0 in basic web app

### ISSUE THREAD

**Issue by TommyLCox** [supabase/supabase-flutter#282]
**Describe the bug**
Basic flutter web app fails at compile startup. 

![image](https://user-images.githubusercontent.com/12648015/203097223-8d55bbcd-7514-4397-a1a6-c58399b77192.png)

The error is in Hive:
![image](https://user-images.githubusercontent.com/12648015/203097488-65f9ef1f-5f57-42df-9e27-a5c9df90ab9d.png)

**To Reproduce**
1. Open Visual Studio Code to a New Folder.
2. CMD/CTRL - SHIFT - P to bring down the command line
3. Type Flutter. Select Flutter: New Project
4. Make sure the device is Chrome(Web). (On Mac, it defaults to Mac OS and runs fine. Need to change to web.)
5. Run the app and make sure it comes up with the counter screen and floating action button and everything builds and runs.
6. Add package supabase_flutter ^1.2.0 to pubspec.
7. Add the initialize to the main function as shown in the image above.
8. Run the application in Chrome. I get the error shown in the image above.

**Expected behavior**.
I expected that supabase would initialize, and the demo program would compile and run normally. 

**Screenshots**
(See above)

**Additional context**
I am having another issue on Supabase Auth in a larger application that seems related to the storage and getting the prior session. I was setting up this simple demo project to demonstrate that error, but got this similar but not identical error. The other one is a SupabaseAuth error and says "no access token found". But, you can tell that it is also using storage. It is identical to #274. I only mention this in case it is relevant that I am having other issues popping up that I haven't been seeing until the last few days.

**Comment by TommyLCox** [comment]
> Thanks @TommyLCox for a detailed steps to reproduce. Unfortunately, I could not reproduce the issue on my end, and I was able to successfully run the initial counter application with supabase-flutter installed. What version of Flutter are you running on your machine?

I am running Flutter 3.3.4. Dart SDK is 2.18.2 (stable).

**Comment by TommyLCox** [comment]
@dshukertjr  - Thank you. You taking the time to run my "Steps to Reproduce" told me that it was possible to get it working. My solution was that I went and deleted from .pub-cache every version of Hive that was cached. I re-ran flutter pub get, and when I did flutter build web, the issue with Hive went away.

Now, the application is producing the error #274 that I originally was trying to replicate from that issue. I will close this and report anything I find with that issue there.
